inc/class-saml.php

```diff
@@ -187,6 +187,24 @@ public function verifyPluginSetup( $options ) {
 		return true;
 	}
 
+	/**
+	 * @param array $net_id
+	 * @param array $attributes
+	 *
+	 * @return string
+	 */
+	public function getEmail( $net_id, $attributes ) {
+		if ( isset( $attributes[ self::SAML_MAP_FIELDS['mail'] ][0] ) ) {
+			$email = $attributes[ self::SAML_MAP_FIELDS['mail'] ][0];
```

Comment:
should this be $attributes[ self::SAML_MAP_FIELDS['mail'] ][1] instead? I'm not sure how the array mapping is supposed to work here.